packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -34,20 +51,54 @@ const RelationInput = ({
   name,
   label,
   labelLoadMore,
-  listHeight,
   loadingMessage,
-  relations,
   onRelationAdd,
   onRelationLoadMore,
-  onSearchClose,
-  onSearchOpen,
   onRelationRemove,
+  onSearchClose,
   onSearchNextPage,
+  onSearchOpen,
   onSearch,
   placeholder,
   publicationStateTranslations,
+  relations,
   searchResults,
 }) => {
+  const listRef = useRef();
+  const [overflow, setOverflow] = useState('bottom');
+
+  // To clean (number relations + numberOfRelationsToDisplay as prop?)
+  const totalNumberOfRelations = relations.data?.pages[0]?.length;
+  const relationItemHeight = 58;
```

Comment:
Could you move this const out of the component and rename it to `RELATION_ITEM_HEIGHT` please?